.github/workflows/ci.yml

```
@@ -277,9 +302,26 @@ jobs:
         working-directory: contracts
         run: forge test
 
+  backend-ci:
+    name: Backend CI (gate)
+    needs: [backend-build, backend, backend-integration]
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check backend CI status
+        run: |
```

actionlint (1.7.12)

[error] got unexpected character '$' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)